Set-1/Solved Programs/KattapaReadyForBattle/TestcaseForKattapa.java

```diff
@@ -0,0 +1,26 @@
+package com.problems;
+
+import static org.junit.Assert.assertEquals;
+import org.junit.Test;
+
+
+public class TestcaseForKattapa {
+
+
+	@Test
```

Comment:
no comments , test name not proper

Reply:
can you suggest any good name for this class.